frontend/app/controllers/sign-in.js

```diff
@@ -0,0 +1,68 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+
+/*---- Login Screen ---*/
+
+  loginFailed: false,
+  loginFailedMessage: "Failed to Login",
+  isProcessing: false,
+  isSlowConnection: false,
+  timeout: null,
+
+    success: function(response) {
```

Comment:
Yes, I've got various sections the open-event code open, but it's not an elementary project, so a certain level of conceptual understanding is required before I can start borrowing from it. thanks.